dtos/v2/account_dto_v2.py

```diff
@@ -0,0 +1,28 @@
+
+
+from dataclasses import dataclass
+import uuid
+
+from models.account_model import Account
+
+
+@dataclass
+class AccountLoginRequestV2(object):
+    email:str
+    password:str
+
+    def __init__(self, request: dict) -> None:
+        self.email = request["email"]
+        self.password = request["password"]
+        super().__setattr__('frozen', True)
```

Comment:
In v2, we moved to a pattern where we used the from_dict method of the dacite library so that specific constructors do not need to be defined.  You can remove this constructor entirely.   To convert from a dict to a dataclass, you would call from_dict(data_class=DTOType, data=dto_dict) (you can reference how it is used in the abstract_create_read_controller_v2.

Reply:
converted